src/server/lib/templateDropdowns.json

```diff
@@ -126,31 +127,31 @@
     "Payments to Individuals"
   ],
   "Sectors Designated as Essential Critical Infrastructure": [
-    "Any work performed by an employee of a State local or Tribal government",
-    "Behavioral health work",
-    "Biomedical Research",
-    "Dental care work",
-    "Educational work school nutrition work and other work required to operate a school facility",
-    "Elections work",
-    "Emergency response",
-    "Family or childcare",
-    "Grocery stores restaurants food production and food delivery",
-    "Health care",
-    "Home- and community-based health care or assistance with activities of daily living",
-    "Laundry work",
-    "Maintenance work",
-    "Medical testing and diagnostics",
-    "Pharmacy",
-    "Public health work",
-    "Sanitation disinfection and cleaning work",
-    "Social services work",
-    "Solid waste or hazardous materials management response and cleanup work",
-    "Transportation and warehousing",
-    "Vital services to Tribes",
-    "Work at hotel and commercial lodging facilities that are used for COVID-19 mitigation and containment",
-    "Work in a mortuary",
-    "Work in critical clinical research development and testing necessary for COVID-19 response",
-    "Work requiring physical interaction with patients",
+    "any work performed by an employee of a State local or Tribal government",
```

Comment:
@jcomeau1 correct that these became lower-case again?

Reply:
I have a note out to the partners to help us verify if the Guidance or the output template is correct in terms of capitalization.